subprojects/composite-builds/src/integTest/groovy/org/gradle/integtests/composite/CompositeBuildIdentityIntegrationTest.groovy

```diff
@@ -243,6 +248,8 @@ Required by:
             }
         """
 
+        executer.expectDocumentedDeprecationWarning(8, "The BuildIdentifier.isCurrentBuild() method has been deprecated. This is scheduled to be removed in Gradle 9.0. Use getBuildPath() to get a unique identifier for the build. Consult the upgrading guide for further information: https://docs.gradle.org/current/userguide/upgrading_version_8.html#build_identifier_name_and_current_deprecation")
```

Comment:
❓ Would it make sense to extract the String to some common class/test fixture so it doesn't have to be repeated in all these places?

Reply:
I would not try to re-use this string.
It is going to be removed in 9.0. It shouldn't be re-used between production and test code, IMO. Sharing it only in the fixtures creates more "infrastructure" around this string than it warrants.